tests/test_click.py

```diff
@@ -3,8 +3,15 @@
 from src.scripts.readactor import cli
 
 
-def test_cli():
+def test_version1():
     runner = CliRunner()
     result = runner.invoke(cli, ["--version"])
     assert result.exit_code == 0
     assert result.output == "version 1.0.1a0"
+
+
+def test_version2():
+    runner = CliRunner()
+    result = runner.invoke(cli, ["-v"])
+    assert result.exit_code == 0
+    assert result.output == "version 1.0.1a0"
```

Comment:
@whalekeykeeper i would make this a regex check, otherwise we ll have to adjust the test every time thee is a version bump. Something like `version {\d+\.}2\d+` same for Line 10 above